### ISSUE TITLE
Face ID is showing a problem when "Application Lock"="No Lock" and...

### ISSUE THREAD

**Issue by juanbretti** [strongbox-password-safe/Strongbox#219]
FaceID is showing a problem when:
```
"Preferences" >> "Application Lock" >> "**No Lock**" 
"Database Management" >> "Face ID Unlock" >> "ON"
```

![faceid IMG_0111](https://user-images.githubusercontent.com/10165911/66985523-3cbcc480-f0bd-11e9-868c-9323732fe956.PNG)

But if I set the locking differently, the error is gone.
```
"Preferences" >> "Application Lock" >> "**Face ID**" 
"Database Management" >> "Face ID Unlock" >> "ON"
```

```
Strongbox version 1.43.3
iOS 13.1.3
```

**Comment by felker** [comment]
I recently encountered this same error when trying to open my database with FaceID, using the first configuration:
```
"Preferences" >> "Application Lock" >> "**No Lock**" 
"Database Management" >> "Face ID Unlock" >> "ON"
```

My setup uses a password + keyfile + HMAC-SHA1 secret (#95) which I entered once and hope to rely on FaceID to bypass it on my iPhone going forward. But after modifying/resaving the database on another system and resyncing it via Google Drive, I ran into this error and was unable to open it on my iPhone with FaceID. 

For me, just restarting the application some hours later made the error go away--- I did not have to change the configuration to use an Application FaceID lock.

**Comment by mmcguill** [comment]
Hi there, definitely seeing issues with Face/Touch ID at the moment but believe this is an iOS bug, due to be fixed in iOS13.2

**Comment by mmcguill** [comment]
Hi @juanbretti - Believe this should be fixed for you now with 1.44.1 - Can you verify and close?

**Comment by juanbretti** [comment]
Perfecto! Thank you for fixing this issue.
